CMakeLists.txt

```diff
@@ -75,7 +70,22 @@ ENDIF()
 
 
 find_program(SED NAMES gsed sed) # gsed - MacOS and FreeBSD, sed - all the rest
-find_file(PCRE_H pcre.h)
+
+pkg_search_module(PCRE REQUIRED libpcre)
+pkg_search_module(JSONC REQUIRED json-c)
+pkg_search_module(LIBGC REQUIRED bdw-gc-threaded bdw-gc)
```

Comment:
I never tested `bdw-gc`. Did you? If not, let's remove it.

Reply:
`bdw-gc` builds with threading by default, only freebsd separates the threaded and non threaded versions.